### PR TITLE
update version to 1.6.1

### PR DESCRIPTION
**Pull request by CharmisG**
Updated IAP flutter Android version to 1.6.2 and iOS version to 1.6.1

### CODE REVIEW

File: example/android/.idea/compiler.xml
Comment (Armaxis) [inline example/android/.idea/compiler.xml:1]: I don't think you should be committing the files from `.idea` folder?

File: README.md [hunk 1]
```diff
@@ -6,8 +6,8 @@
 The Flutter plugin for Square [In-App Payments SDK] is a wrapper for the native Android and iOS SDKs and 
 supports the following native In-App Payments SDK versions:
 
-  * iOS: 1.5.7
-  * Android: 1.5.6
+  * iOS: 1.6.0
+  * Android: 1.6.1
```
Comment (Armaxis) [inline README.md:10]: Android should be `1.6.2` and iOS - `1.6.1`, please update these versions across the PR

File: example/android/.idea/compiler.xml
Comment (CharmisG) [reply example/android/.idea/compiler.xml:1]: Removed all .idea files !

**Review by Armaxis — APPROVED** [review]
Android changes look good!

**Comment by brandonjenniges** [comment]
iOS changes look good

**Review by brandonjenniges — APPROVED** [review]

**Review by brandonjenniges — APPROVED** [review]

**Review by Armaxis — APPROVED** [review]

**Review by fka3 — APPROVED** [review]

**Review by WhatsEmo — APPROVED** [review]

**Pull request merged**